### PR TITLE
Update cla in README

### PR DESCRIPTION
**Pull request by knightast**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,74 +10,66 @@ Functionalities:
 - Labeled property graph model
 - Full ACID support with serializable transactions
 - Graph analytics algorithms embedded with graph computing framework
-- OpenCypher query language
-- Primary/Secondary Index in vertex and edge
-- Full-Text Index support
-- Graph visualization with web interface
-- Java/Python/C++ clients available
-- RESTful and RPC API support
+- Full-Text/Primary/Secondary/Full-Text Index support
```
Comment (hjk41) [inline README.md:13]: Full-text写了两遍

File: README.md [hunk 2]
```diff
@@ -10,74 +10,66 @@ Functionalities:
...
+
+Welcome to our [website](https://www.tugraph.org)
 
 ## 1. Quick Start
 
 An easy way to start is using docker to set up, which can be found in DockerHub, named `tugraph/tugraph-db-[os]:[tugraph version]`,
 for example, `tugraph/tugraph-db-centos7:3.3.0`.
 
-For more details, please refer to [doc-zh/1.guide/3.quick-start.md]
+For more details, please refer [quick start doc](doc/zh-CN/1.guide/3.quick-start.md).
```
Comment (hjk41) [inline README.md:31]: refer -> refer to

File: README.md [hunk 3]
```diff
@@ -10,74 +10,66 @@ Functionalities:
...
 
 ## 2. Install
 
-### 2.1 Compile the project with GCC on UNIX:
+We suggest to compile TuGraph in UNIX system, docker environment is a good choice. If you want to setup a new environment, please refer [Dockerfile](ci/images).
```
Comment (hjk41) [inline README.md:35]: Install -> Building From Source

UNIX -> Linux

Comment (hjk41) [reply README.md:35]: We suggest 这句很中文，英文一般用被动式
It is recommended to build...

File: README.md [hunk 4]
```diff
@@ -10,74 +10,66 @@ Functionalities:
...
+
+Here's steps to compile TuGraph:
```
Comment (hjk41) [inline README.md:37]: Here's -> Here are the

File: README.md (hunk 1 above)
Comment (knightast) [reply README.md]: DONE

File: README.md (hunk 2 above)
Comment (knightast) [reply README.md]: DONE

File: README.md (hunk 3 above)
Comment (knightast) [reply README.md]: DONE

File: README.md (hunk 4 above)
Comment (knightast) [reply README.md]: DONE

**Review by hjk41 — APPROVED** [review]

**Pull request merged**